EngineLayer/PeptideSpectralMatch.cs

```diff
@@ -26,10 +26,11 @@ public PeptideSpectralMatch(PeptideWithSetModifications peptide, int notch, doub
             ScanPrecursorCharge = scan.PrecursorCharge;
             ScanPrecursorMonoisotopicPeakMz = scan.PrecursorMonoisotopicPeakMz;
             ScanPrecursorMass = scan.PrecursorMass;
-            DigestionParams = digestionParams;
+            DigestionParams = commonParameters.DigestionParams;
             PeptidesToMatchingFragments = new Dictionary<PeptideWithSetModifications, List<MatchedFragmentIon>>();
             Xcorr = xcorr;
             NativeId = scan.NativeId;
+            RunnerUpScore = commonParameters.ScoreCutoff;
```

Comment:
<3

Reply:
?

Reply:
no, it was a heart emoji. Sorry for the confusion. The current code is good.